spa/cypress/integration/11 - dashboard.spec.js

```diff
@@ -5,32 +5,43 @@ import { DASHBOARD_SLUG, DONATIONS_SLUG, CONTENT_SLUG } from 'routes';
 import hubAdminWithoutFlags from '../fixtures/user/hub-admin';
 import {
   CONTRIBUTIONS_SECTION_ACCESS_FLAG_NAME,
+  CONTRIBUTIONS_SECTION_DENY_FLAG_NAME,
   CONTENT_SECTION_ACCESS_FLAG_NAME
 } from 'constants/featureFlagConstants';
 
-const contribSectionsFlag = {
+const contribSectionsAccessFlag = {
   id: '1234',
   name: CONTRIBUTIONS_SECTION_ACCESS_FLAG_NAME
 };
 
+const contribSectionsDenyFlag = {
+  id: '1235',
+  name: CONTRIBUTIONS_SECTION_DENY_FLAG_NAME
+};
+
 const contentSectionFlag = {
   id: '5678',
   name: CONTENT_SECTION_ACCESS_FLAG_NAME
 };
 
-const hubAdminWithContributionsFlag = {
+const hubAdminWithContributionsAccessFlag = {
   ...hubAdminWithoutFlags,
-  flags: [{ ...contribSectionsFlag }]
+  flags: [{ ...contribSectionsAccessFlag }]
```

Comment:
No need to deconstruct object into a new object
```suggestion
  flags: [contribSectionsAccessFlag]
```

Reply:
Addressed here 3beadee40bae770316d5ca5dc8e0e2a066e53da8